src/api/custom-resolvers/mutations/create-an-account/account-exists-blocked.test.ts

```diff
@@ -0,0 +1,252 @@
+import createAnAccount from '.';
+import { ACCOUNT } from '../../../constants';
+import sendEmail from '../../../emails';
+import confirmEmailAddressEmail from '../../../helpers/send-email-confirm-email-address';
+import accounts from '../../../test-helpers/accounts';
+import accountStatus from '../../../test-helpers/account-status';
+import { mockAccount, mockSendEmailResponse } from '../../../test-mocks';
+import { Account, Context } from '../../../types';
+import getKeystoneContext from '../../../test-helpers/get-keystone-context';
+import sendEmailReactivateAccountLinkHelper from '../../../helpers/send-email-reactivate-account-link';
+import crypto from 'crypto';
+
+const { ENCRYPTION } = ACCOUNT;
+
+const {
+  PASSWORD: {
+    PBKDF2: { KEY_LENGTH },
+  },
+  STRING_TYPE,
+  PBKDF2: { ITERATIONS, DIGEST_ALGORITHM },
+} = ENCRYPTION;
+
+describe('custom-resolvers/create-an-account', () => {
```

Comment:
Can we update this with a description at the end "account exists and is blocked"?